book/eha.qmd

```diff
@@ -11,25 +11,27 @@ abstract: TODO (150-200 WORDS)
 |-|-|-|
-| one/first event | single event | competing risks |
-| multiple events | recurrent events | multi state |
+| One/first event | single event | competing risks |
+| Multiple events | recurrent events | multi state |
+: Classifying events in event-history analysis. Top row refers either to the setting where only one event occurs, or the setting where multiple events occur but only the time to the first event is of interest. {#tbl-eha}
 
-Another way to think about event history is in terms of transitions between different states, as illustrated in @fig-eha.
-Usually, a subject starts out in state 0 (for example, healthy) and transitions to different states from there.
+One can think about event history in terms of transitions between different states, as illustrated in @fig-eha.
+Usually, a subject starts out in state 0 (for example, 'healthy') and transitions to different states from there.
 States from which further transitions are possible are called *transient*, otherwise a state is called *terminal*.
 If no transition happens within the follow-up time, the subject remains in the initial state, in event history terms the observation is censored for this transition.
 
-In the *single-event setting* (@fig-eha, left panel), a subject can only transition to one state (the event of interest).
-In the *competing risks setting* (@fig-eha, middle panel, see @sec-competing-risks), a subject could transition to any of the $K$ mutually exclusive states, thus the subject is *at risk* for a transition to multiple states.
+In the *single event* setting (@fig-eha, left panel), a subject can only transition to one state (the event of interest).
+In the *competing risks* setting (@fig-eha, middle panel, see @sec-competing-risks), a subject could transition to any of the $K$ mutually exclusive states, thus the subject is *at risk* for a transition to multiple states.
 However, once one of them occurs, it is impossible to transition to another (for example, death from different causes).
 In both cases, the event needs not be terminal, we just end the observation after the first occurence of an event.
 
-In the most general case, the *multi-state setting* (@fig-eha right panel, see @sec-multi-state), there are multiple transient and terminal states with potential back transitions (for example, moving between different stages of an illness with the possibility of (partial) recovery and death as terminal event).
+In the most general case, the *multi-state* setting (@fig-eha right panel, see @sec-multi-state), there are multiple transient and terminal states with potential back transitions (for example, moving between different stages of an illness with the possibility of (partial) recovery and death as terminal event).
 The *recurrent events setting* is as a special case of the multi-state setting but could also be treated as a more general case of the single event setting with correlated observations (@sec-recurrent-events and @fig-recurrent-events) .
```

Comment:
I tried to fix myself but got very confused. What's the middle panel? Confused between competing risks and competing events. Jumps between explaining columns and plots. I think it would be easiest just to explain each panel separately and maybe start or finish that section by saying what groups rows and columns